src/GafferImage/ImageWriter.cpp

```diff
@@ -705,7 +705,7 @@ ImageWriter::ImageWriter( const std::string &name )
 	storeIndexOfNextChild( g_firstPlugIndex );
 	addChild( new ImagePlug( "in" ) );
 	addChild( new StringPlug( "fileName" ) );
-	addChild( new StringPlug( "channels", Gaffer::Plug::In, "[RGB]" ) );
+	addChild( new StringPlug( "channels", Gaffer::Plug::In, "*" ) );
```

Comment:
This seems a much better default, so id be happy to accept the breaking change. I've found it quite fiddly to have to remember to add A manually thus far.